person.rb

```diff
@@ -1,9 +1,50 @@
-class Person
-  def initialize(name: 'Unknown', age: nil, parent_permission: true)
+require_relative 'Nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameable.correct_name.capitalize
```

Comment:
- [ ] just as the previous reviewer mentioned it should be ` @nameable.correct_name` not ` @nameable.correct_name.capitalize`. kindly fix that